pyresample/spherical.py

```diff
@@ -34,22 +34,121 @@ def _unwrap_radians(val, mod=np.pi):
     return (val + mod) % (2 * mod) - mod
 
 
+def _xyz_to_vertices(x, y, z):
+    """Create vertices array from x,y,z values or vectors.
+
+    If x, y, z are np.values, it create a 1x3 np.array.
```

Comment:
Is the other common term for this a "scalar array"?

Reply:
```suggestion
    If x, y, z are scalar arrays, it creates a 1x3 np.array.
```